Description: build s390, armel, mips* debugging symbols with -gstabs
 -g exhausts memory on these arches. Thanks to Oswald Buddenhagen for the hint.
Author: Sune Vuorela <sune@debian.org>
Author: Modestas Vainius <modax@debian.org>
Bug-Debian: http://bugs.debian.org/528485
Bug-Debian: http://bugs.debian.org/618325

---
 Source/WebKit.pri |    7 +++++++
 1 file changed, 7 insertions(+)

--- a/Source/WebKit.pri
+++ b/Source/WebKit.pri
@@ -80,6 +80,13 @@ CONFIG(release, debug|release) {
     DEFINES += NDEBUG
 }
 
+equals(QT_ARCH, s390)|equals(QT_ARCH, arm)|equals(QT_ARCH, mips) {
+	message("s390, armel, armhf, mipsel workaround for QtWebkit: do not build with -g, but with -gstabs")
+	QMAKE_CXXFLAGS -= -g
+	QMAKE_CXXFLAGS += -gstabs
+}
+
+
 INCLUDEPATH += \
     $$PWD \
     $$PWD/../include \
